frontend/pages/packs/EditPackPage/EditPackPage.tsx

```diff
@@ -0,0 +1,363 @@
+    select: (data: IStoredFleetQueriesResponse) => data.queries,
+  });
+
+  const { data: labels } = useQuery<IStoredLabelsResponse, Error, ILabel[]>(
+    ["pack labels"],
+    () => labelsAPI.loadAll(),
+    {
+      select: (data: IStoredLabelsResponse) => data.labels,
+    }
+  );
+
+  const packLabels = storedPack
+    ? filter(labels, (label) => {
+        return includes(storedPack.label_ids, label.id);
+      })
+    : [];
+
+  const { data: hosts } = useQuery<IStoredHostsResponse, Error, IHost[]>(
+    ["all hosts"],
+    () => hostsAPI.loadAll({ perPage: 30000 }),
+    {
+      select: (data: IStoredHostsResponse) => data.hosts,
+    }
+  );
+
+  const packHosts = storedPack
+    ? filter(hosts, (host) => {
+        return includes(storedPack.host_ids, host.id);
+      })
+    : [];
+
+  const { data: teams } = useQuery<IStoredTeamsResponse, Error, ITeam[]>(
+    ["all teams"],
+    () => teamsAPI.loadAll(),
+    {
+      select: (data: IStoredTeamsResponse) => data.teams,
+    }
+  );
+
+  const packTeams = storedPack
+    ? filter(teams, (team) => {
+        return includes(storedPack.team_ids, team.id);
+      })
+    : [];
+
+  const packTargets = [
+    ...packHosts.map((host) => ({
+      ...host,
+      target_type: "hosts",
+    })),
+    ...packLabels,
+    ...packTeams.map((team) => ({
+      ...team,
+      target_type: "teams",
+      display_text: team.name,
+    })),
+  ];
+
+  const onCancelEditPack = () => {
+    return dispatch(push(PATHS.MANAGE_PACKS));
+  };
+
+  const onFetchTargets = useCallback(
+    (query: IQuery, targetsResponse: ITargetsAPIResponse) => {
+      const { targets_count } = targetsResponse;
+      setTargetsCount(targets_count);
+
+      return false;
+    },
+    []
+  );
+
+  const togglePackQueryEditorModal = () => {
+    setSelectedPackQuery(undefined); // create modal renders
+    setShowPackQueryEditorModal(!showPackQueryEditorModal);
+  };
+
+  const toggleRemovePackQueryModal = () => {
+    setShowRemovePackQueryModal(!showRemovePackQueryModal);
+  };
+
+  const onEditPackQueryClick = (selectedQuery: IScheduledQuery): void => {
+    togglePackQueryEditorModal();
+    setSelectedPackQuery(selectedQuery); // edit modal renders
+  };
+
+  const onRemovePackQueriesClick = (selectedTableQueryIds: number[]): void => {
+    toggleRemovePackQueryModal();
+    setSelectedPackQueryIds(selectedTableQueryIds);
+  };
+
+  // If we want to use a try/catch pattern in future iterations
+  // const updatePack = useCallback(
+  //   async (updatedPack) => {
+  //     setIsStoredPackLoading(true);
+  //     try {
+  //       const response = await packsAPI.update(packId, updatedPack);
+  //       console.log(response);
+  //       setStoredPack(response.pack);
+  //     } catch (error) {
+  //       console.log(error);
+  //       setIsStoredPackLoadingError(true);
+  //     } finally {
+  //       setIsStoredPackLoading(false);
+  //     }
+  //   },
+  //   [dispatch]
+  // );
```

Comment:
Is the commented code waiting for green light to complete? If not, let's remove it.